src/act-runner/runTestsInPage.ts

```diff
@@ -17,6 +18,29 @@ async function startPuppeteer(): Promise<{page: Page, browser: Browser}> {
   const browser = await puppeteer.launch();
   const page = await browser.newPage();
   await page.setBypassCSP(true);
+  await page.setRequestInterception(true);
+
+  // Chrome has no option to prevent meta refresh
+  // So instead, we'll tweak any examples that has immediate refresh / redirects
+  page.on("request", async request => {
+    const url = request.url();
+    // Rules that have meta[refresh] elements
+    if (!url.includes(`/bc659a/`) && !url.includes(`/bisz58/`)) {
+      request.continue();
+      return
+    }
+    try {
+      let body = (await Axios.get<string>(url)).data;
+      if (body.match(/content="\s*0+(;.*"|")/)) {
+        body = body.replace(/content=".*"/g, 'content=""');
```

Comment:
This is a really board stroke and the replace will replace more than just the content attribute. If two elements are on the same line it would get everything to the last `"`

```html
<meta content="  0;"><div class="hello"></div>
```

At the very least should add a `?` to the `*` matcher to get the smallest match. The general matching of any `content` attribute has the potential to cause problems (especially if there are other `meta` tags on the page), but since `http-equiv` is about the only one that will start with a 0, should be fine. 

```suggestion
      if (body.match(/content="\s*0+(;.*?"|")/)) {
        body = body.replace(/content=".*?"/g, 'content=""');
```